svelte/src/lib/dialogs/DeleteTestCaseDialog.svelte

```diff
@@ -0,0 +1,151 @@
+<script lang="ts">
+  import { combinePath } from "$lib/components/FileExplorer/Node";
+  import { Notify } from "$lib/components/Notify";
+  import PrimaryButton from "$lib/components/PrimaryButton.svelte";
+  import StandardButton from "$lib/components/StandardButton.svelte";
+  import { AppActionType, appContextKey, type IAppContext } from "$lib/context/AppContext";
+  import { stringResKeys } from "$lib/context/StringResKeys";
+  import { uiContextKey, type IUiContext } from "$lib/context/UiContext";
+  import { fileSystem } from "$lib/ipc";
+  import { StandardFolder, type ITestCase, type ITestSuite } from "rockmelonqa.common/file-defs";
+  import { getContext, onDestroy, onMount } from "svelte";
+
+  /** Toggle on dialog */
+  export let showDialog: boolean = false;
+  /** File system physical path */
+  export let testCaseFilePath: string;
+  /** Path on tree (used for closing coresponding tab after deleting testcase node) */
+  export let nodePath: string;
+  /** Array of relative paths of the test suites that contains the test case being deleted */
```

Comment:
'Array of test suite relative paths, which contain the test case being deleted'

Reply:
Will be fixed when we move the "delete" logic to Node.svelte